appchain/src/lib.rs

```diff
@@ -1085,7 +1074,7 @@ pub mod pallet {
 			Ok(())
 		}
 
-		#[pallet::weight(0)]
+		#[pallet::weight(<T as Config>::WeightInfo::force_set_next_notification_id(*next_notification_id))]
```

Comment:
Why does `next_notification_id` affect the weight?

Reply:
This parameter is required here because it is generated randomly in benchmarking. In fact, it does not work when calculating weights.    Maybe you can see here, https://github.com/octopus-network/octopus-pallets/blob/andy/fix/add-weight/appchain/src/weights.rs#L280

Reply:
OK, fixed.